src/app/services/authentication.service.ts

```diff
@@ -30,24 +30,26 @@ const auth0CordovaConfig : IonicAuthOptions = {
   iosWebView: 'private'
 };
 
-const auth0WebConfig : IonicAuthOptions = {
+const auth0WebConfig: IonicAuthOptions = {
   // the auth provider
   authConfig: 'auth0',
   // The platform which we are running on
   platform: 'web',
   // client or application id for provider
-  clientID: 'ihSRqLLa2z33PTyeNNlI2uxgsqorb08l',
+  clientID: 'vbOi3NDg3eRfQTUL4FuTqhaQHcHKPgOh',
   // the discovery url for the provider
   // OpenID configuration
-  discoveryUrl: 'https://ionicorg.auth0.com/.well-known/openid-configuration',
+  discoveryUrl: 'https://ionicteam.auth0.com/.well-known/openid-configuration',
```

Comment:
is this the same org? I set up a special user account for Auth0 sign-in - would want to ensure that everything's configured correctly for web and mobile. yea?

Reply:
I don't think I have access to `ionicorg` on Auth0, so I set up a new app in the Ionic team.